roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -53,7 +55,7 @@ kafka_broker:
   authorizer_appender_max_log_files: 10
   authorizer_appender_log_file_size: 100MB
   datadir:
-    - /var/lib/kafka/data
```

Comment:
I wonder if this change is really necessary, could be breaking for people

Reply:
seems it is necessary... what are the implications?
I guess if someone reruns all.yml after this the data dir will be wrong, but we dont support reconfiguration yet...

Reply:
It's definitely a breaking change if someone runs the playbook again, against a working cluster.  It will cause them to potentially lose data, as it moves the data dir, and if they don't realize that for a while, they will then be missing historical data, and trying to merge that together is very ugly if not practical/possible.   I think if we want to make this change, we should do it as part of a major release and  Bold it in the documentation and release notes.

Reply:
I found a way to not update the data dir, for whatever reason, the user needs to own /var/lib/kafka regardless or what the data dir is, see in my latest comment. I would say though itd be nice to get back to the default, but i agree that would disrupt customers, not sure how worth it it is

Reply:
@domenicbove so is the plan to leave as is, and then we can change it in the next major release?

Reply:
I do not have a plan to change it, i figured we'd have to make that together. right now this PR works and is non disruptive wrt kafka data dir